serverless.yml

```diff
@@ -107,3 +120,7 @@ functions:
             parameters: 
               paths: 
                 hash: true
+          caching:
```

Comment:
Isn't `cors: true` needed anymore ??

Reply:
Good catch, it went missing for some reason.

Besides, the frontend is indeed getting a cors error.
I'll readd this and test again.